### PR TITLE
Updated Graph UI

### PR DESCRIPTION
**Pull request by pramit-d**
https://github.com/Be-Secure/BeSLighthouse/issues/509 
1. Reduced node size.
2. Changed model node color (black and orange).
3. Improved graph UI by adding force between the node.

Fixed Bugs: https://github.com/Be-Secure/BeSLighthouse/issues/507 https://github.com/Be-Secure/BeSLighthouse/issues/506

### CODE REVIEW

File: src/pages/ModelOfInterest/GraphDisplay/index.tsx [hunk 1]
```diff
@@ -64,53 +64,39 @@ const GraphDisplay = () => {
...
     // Create a simulation with several forces.
     const simulation = d3.forceSimulation(nodes)
-      .force("link", d3.forceLink(links).id((d) => (d as Node).name).distance(50))
-      .force("charge", d3.forceManyBody().strength(-20))
-      .force("center", d3.forceCenter(width / 2, height / 2))
+      .force("link", d3.forceLink(links).id((d) => (d as Node).name))
+      .force("charge", d3.forceManyBody().strength(-250))
+      .force("x", d3.forceX())
+      .force("y", d3.forceY())
+      //.force("link", d3.forceLink(links))
```
Comment (harimohanr) [inline src/pages/ModelOfInterest/GraphDisplay/index.tsx:73]: @pramit-d Please remove the commented code blocks

File: src/pages/ModelOfInterest/GraphDisplay/index.tsx [hunk 2]
```diff
@@ -64,53 +64,39 @@ const GraphDisplay = () => {
...
         // Create the SVG container.
         const svg = d3.select("#graph-container")
           .append("svg")
-          .attr("width", "100%")
-          .attr("height", "100%")
-          .attr("viewBox", `0 0 ${width} ${height}`)
-          .append("g")
-          .attr("transform", `translate(${margin.left},${margin.top})`);
+          //.attr("width", "100%")
+          //.attr("height", "100%")
```
Comment (harimohanr) [inline src/pages/ModelOfInterest/GraphDisplay/index.tsx:81]: @pramit-d Please removed these commented code blocks

**Review by harimohanr — APPROVED** [review]
@pramit-d Looks okay to me

File: src/pages/ModelOfInterest/GraphDisplay/index.tsx (hunk 1 above)
Comment (pramit-d) [reply src/pages/ModelOfInterest/GraphDisplay/index.tsx]: @harimohanr  removed commented code.

**Pull request merged**